### PR TITLE
Add telephone appointment feature toggle for check-in application

### PR DESCRIPTION
**Pull request by brdunfield**
## Description
This PR adds the
`check_in_experience_phone_appointments_enabled` feature toggle to the app. The toggle is currently non functional.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#40839

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by it-harrison — APPROVED** [review]

**Review by acrollet — CHANGES_REQUESTED** [review]
This looks good! For consistency we should document this flag in the day-of and pre-checkin READMEs as well but otherwise g2g.

**Review by it-harrison — APPROVED** [review]

File: src/applications/check-in/day-of/README.md [hunk 1]
```diff
@@ -152,6 +152,8 @@ Though we have the HOC, its now considered best practice to query redux using th
   - when to sunset: once we have successfully tested this feature in production with users and the backend has fully switched over
 - `check_in_experience_edit_messaging_enabled` : Enables or disables edit messaging.
   - when to sunset: once we have successfully tested this feature in production with users and the backend has fully switched over
+- `check_in_experience_phone_appointments_enabled` : Enables or disables telephone appointments as an alternate type to in-person
+  - when to sunset: once we have successfully tested this geature in production with users
```
Comment (acrollet) [inline src/applications/check-in/day-of/README.md:156]:
```suggestion
  - when to sunset: once we have successfully tested this feature in production with users
```

File: src/applications/check-in/pre-check-in/README.md [hunk 2]
```diff
@@ -101,6 +101,8 @@ Though we have the HOC, its now considered best practice to query redux using th
   - when to sunset: once we have successfully tested this feature in production with users and the backend has fully switched over
 - `check_in_experience_edit_messaging_enabled` : Enables or disables edit messaging.
   - when to sunset: once we have successfully tested this feature in production with users and the backend has fully switched over
+- `check_in_experience_phone_appointments_enabled` : Enables or disables telephone appointments as an alternate type to in-person
+  - when to sunset: once we have successfully tested this geature in production with users
```
Comment (acrollet) [inline src/applications/check-in/pre-check-in/README.md:105]:
```suggestion
  - when to sunset: once we have successfully tested this feature in production with users
```

**Review by acrollet — APPROVED** [review]
added typo suggestions but looks good! 🚢

**Pull request merged**